qa-pipelines/tasks/cf-deploy.sh

```diff
@@ -94,4 +116,23 @@ if [[ ${cap_platform} =~ ^azure$|^gke$|^eks$ ]]; then
     azure_set_record_sets_for_namespace scf
 fi
 
+if pxc_pre_upgrade; then
+    echo "Downsizing SCF mysql node count to 1..."
+    echo "${HELM_PARAMS[@]}" | sed 's/kube\.registry\.password=[^[:space:]]*/kube.registry.password=<REDACTED>/g'
+    helm upgrade scf ${CAP_DIRECTORY}/helm/cf/ \
+        --namespace "${CF_NAMESPACE}" \
+        --timeout 3600 \
+        --set "secrets.CLUSTER_ADMIN_PASSWORD=${CLUSTER_ADMIN_PASSWORD:-changeme}" \
+        --set "env.UAA_HOST=${UAA_HOST}" \
+        --set "env.UAA_PORT=${UAA_PORT}" \
+        --set "secrets.UAA_CA_CERT=${CA_CERT}" \
```

Comment:
HELM_PARAMS contains the correct value for this, but `CA_CERT` isn't set and causes an error